maro/rl/training/algorithms/ddpg.py

```diff
@@ -40,6 +37,7 @@ class DDPGParams(TrainerParams):
     q_value_loss_cls: Callable = None
     soft_update_coef: float = 1.0
     random_overwrite: bool = False
+    n_start_train: int = 0
```

Comment:
renamed to sth. like `min_num_to_trigger_training` ?

Reply:
What's the relationship between this and `min_n_sample` in main.py? If they are for the same purpose, can we remove this here?